crates/onnx-ir/src/node/prelu.rs

```diff
@@ -25,18 +25,25 @@
 //! - The slope input is lifted to static during constant lifting phase
 //! - This allows the slope to be embedded in the generated code
 
-use crate::ir::{Argument, Node, RawNode};
+use crate::ir::{ArgType, Argument, Node, RawNode};
 use crate::processor::{
     InputSpec, NodeProcessor, NodeSpec, OutputPreferences, OutputSpec, ProcessError,
 };
 use onnx_ir_derive::NodeBuilder;
 
+#[derive(Debug, Clone, new)]
+pub struct PReluConfig {
+    /// Number of learnable parameters
+    pub num_parameters: usize,
+}
+
 /// Node representation for PRelu operation
 #[derive(Debug, Clone, NodeBuilder)]
 pub struct PReluNode {
     pub name: String,
     pub inputs: Vec<Argument>,
     pub outputs: Vec<Argument>,
+    pub config: PReluConfig,
```

Comment:
`PReluConfig` should be removed.

Node configuration is meant to more ONNX configuration rather Burn's module configurations. 

ONNX-IR is a higher abstraction for ONNX graph. Meaning anyone (outside Burn) can also consume.